config/software/openssl.rb

```diff
@@ -32,7 +32,7 @@
 dependency "cacerts"
 dependency "openssl-fips" if fips_mode?
 
-default_version "1.1.1q" # do_not_auto_update
+default_version "1.1.1t" # do_not_auto_update
```

Comment:
This did not actually add the checksum below, which broke our build…

Reply:
Opened #404 to resolve.